src/main/java/ca/gc/cyber/ops/assemblyline/java/client/clients/AssemblylineClient.java

```diff
@@ -530,6 +550,13 @@ private BodyInserters.MultipartInserter multipartInserterFromBinaryIngest(Binary
         return BodyInserters.fromMultipartData(mbb.build());
     }
 
+    private BodyInserters.MultipartInserter multipartInserterFromAsyncBinaryIngest(AsyncBinaryFile<?> binaryFile) throws JsonProcessingException {
+        MultipartBodyBuilder mbb = new MultipartBodyBuilder();
+        mbb.asyncPart(MULTIPART_MSG_BINARY_PART, binaryFile.getFile(), ByteBuffer.class).filename(binaryFile.getFilename()).contentType(MediaType.APPLICATION_OCTET_STREAM);
```

Comment:
This is a fairly long line. I think breaking it up would be more legible.

```suggestion
        mbb.asyncPart(MULTIPART_MSG_BINARY_PART, binaryFile.getFile(), ByteBuffer.class)
            .filename(binaryFile.getFilename())
            .contentType(MediaType.APPLICATION_OCTET_STREAM);
```